pkg/dmsg/client.go

```diff
@@ -74,6 +74,10 @@ type Client struct {
 	conf   *Config
 	porter *netutil.Porter
 
+	bo     time.Duration // initial backoff duration
```

Comment:
Can we use the `netutil.Retrier` here?

Reply:
`netutil.Retrier` doesn't work in this case as we need to wait with exponential backoff in multiple places. And to do that in the retrier we have to return an error but by doing that we have to restart the [main loop](https://github.com/ersonp/dmsg/blob/d639576132dde18673c20851018045ac2a960157/pkg/dmsg/client.go#L152) all over again instead of the current version where we wait between each entry [here](https://github.com/ersonp/dmsg/blob/d639576132dde18673c20851018045ac2a960157/pkg/dmsg/client.go#L172).